worker/categorize.py

```diff
@@ -101,9 +101,8 @@ def categorize_item(self, title, text):
 
         # Build a graph and identify keyword clusters
         clustering = None
-
-        if len(assigned_articles) > 2:
-            semantic_graph = self._build_semantic_graph(assigned_articles)
+        semantic_graph = self._build_semantic_graph(assigned_articles)
+        if semantic_graph.edges():
```

Comment:
I didn't do feed_seed to test, but I wrote a test that was returning the dendogram error and added this code to get it to pass.

I removed the test because I didn't feel it was necessary to keep once we fixed it.